test/index.js

```diff
@@ -188,79 +189,13 @@ describe('big-json', function() {
             parseStream.on('error', function(err) {
                 assert.ok(err);
                 assert.equal(err.name, 'Error');
-                assert.equal(err.message, 'Parser has expected a string value');
+                assert.equal(err.message, 'Invalid JSON (Unexpected "\\n" ' +
```

Comment:
Should we couple our tests to the error message? Is having an error with a message enough?

Reply:
Shortened to test just for 'Invalid JSON'. Coupling isn't great, but in this case I do want to verify that it's because of invalid content vs some other internal error.